src/microbe_stage/MicrobeStage.cs

```diff
@@ -343,7 +355,7 @@ public void OnFinishLoading()
 
     public void StartNewGame()
     {
-        CurrentGame = GameProperties.StartNewMicrobeGame();
+        CurrentGame = GameProperties.StartNewMicrobeGame(WorldSettings);
```

Comment:
This should just create a new world settings as this is called (I think) only when playing the scene directly from Godot.

Reply:
I discovered while testing InitialWorldSettings that we do need to pass the settings object here, even in the normal game mode. So I've kept it.

Reply:
I think this is related to: https://github.com/Revolutionary-Games/Thrive/pull/3389#discussion_r902677928